config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,19 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// BareMetalPlatformStatus holds the current status of the BareMetal infrastructure provider.
+type BareMetalPlatformStatus struct {
+	// apiVIP holds the Virtual IP (VIP) that can be used to contact the external OpenShift API
+	APIVIP string `json:"apiVIP"`
+
+	// ingressVIP holds the Virtual IP (VIP) for the Ingress operator
+	IngressVIP string `json:"ingressVIP"`
```

Comment:
@ironcladlou you aware of this and is is the doc correct?

Reply:
Who sets/manages this field, and who consumes it?

Reply:
This field is described in terms of the ingress operator and ingress controllers, but doesn't seem to align precisely with the design of ingress:

* There are one or more ingress controllers (routers) in a cluster — the default one, and any user defined ones.
* Individual ingress controllers are reachable through `.status.domain`, so even assuming this refers to a specific ingress controller (the default one?), it's not clear from what the IP field is derived.

I could use some more context about what exactly this field is intended to represent and how it should relate to ingress.

Reply:
The field is set in the installer, and will be consumed by MCO and perhaps baremetal-operator/cluster-api-provider-baremetal.

I will add detail about what specific ingress router this refers to.

Reply:
>The field is set in the installer, and will be consumed by MCO and perhaps baremetal-operator/cluster-api-provider-baremetal.

>I will add detail about what specific ingress router this refers to.

If the value isn't consumed by ingress-operator, in what way does the field relate to ingress controllers (which are implemented by ingress-operator)?

Reply:
This IP address is a configuration item for IPI bare metal clusters.  The IP is reserved for the cluster's use.  Specifically, it's an IP that will always reside on a host where one of the routers is running.  This is what the apps DNS record would be pointing to by default.  Customers can then optionally, set up an external load balancer in front of multiple routers if they choose to.